pkg/manager/member/template.go

```diff
@@ -177,7 +177,7 @@ fi {{- else}}{{.CheckDomainScript}}{{end}}
 done
 
 ARGS="--data-dir={{ .DataDir }} \
---name={{- if .ClusterDomain }}${domain}{{- else }}${POD_NAME}{{- end }} \
+--name={{- if .AcrossK8s }}${domain}{{- else }}${POD_NAME}{{- end }} \
```

Comment:
I think this should not be changed because the cluster domain determines which address to use instead of acrossK8s.

Reply:
@KanShiori a bug is introduced because of it = =